lib/adapter/adapters/memory/index.js

```diff
@@ -26,7 +26,8 @@ export default Adapter => class MemoryAdapter extends Adapter {
   }
 
 
-  find (type, ids, options = {}) {
+  find (type, ids, options) {
+    if (!options) options = {}
```

Comment:
I'm pretty sure I tried that and it also caused test failures
On Aug 20, 2015 12:40 AM, "0x8890" <notifications@github.com> wrote:

> In lib/adapter/adapters/memory/index.js
> <https://github.com/fortunejs/fortune/pull/117#discussion_r37499114>:
>
> > @@ -26,7 +26,8 @@ export default Adapter => class MemoryAdapter extends Adapter {
> >    }
> >
> >
> > -  find (type, ids, options = {}) {
> > +  find (type, ids, options) {
> > +    if (!options) options = {}
>
> Actually I think it is safe to not assign a default value for options.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fortunejs/fortune/pull/117/files#r37499114>.
>